### DISCUSSION TITLE
Removing the "generated:" timestamp from the class files

### DISCUSSION THREAD

**Discussion by jrebs** [netsuitephp/netsuite-php#93] [Ideas]
For @neclimdul, @ryanwinchester or anyone else who cares to comment:

Is there any real value in continuing to populate the "generated:" line that we currently inject into the class level docblocks when the class separator generates a new set of class files? It seems to me that since we are using revision control, we can pretty easily look, if needed, to see when these files were generated. I think the real question anyone might have while looking at a file is "which version of the toolkit was this generated against?" and that question isn't really answered by the generated timestamp anyway. For this information, a user could look at the `$generated_from_endpoint` in the `NetSuiteService` class.

The only thing I see the generated datetime line doing is making it damn near impossible to evaluate a new build's differences and quickly see which class files were actually changed. Every time we build a new version of the toolkit, we end up with nearly 2000 changed files just for these timestamps, when in practice often only a dozen or so files might have any real changes.

Unless someone can point out that I'm missing some valuable reason to keep the line there, I'm planning to make an update prior to the `v2021.1.0` that will remove this line from the files. Bueller?

**Comment by ryanwinchester** [comment]
In my opinion, I'd remove it from the `NetSuite\Classes\*` namespaced classes and just keep it in the `NetSuiteService` and/or `NetSuiteClient` classes (if at all).

**Reply by jrebs** [reply]
Perfect solution. I like the idea of putting this in the NetSuiteService, as it's not a class provided in the toolkit.

@ryanwinchester while I've got your attention, some files have a header that reads:

`This file is part of the SevenShores/NetSuite library AND originally from the NetSuite PHP Toolkit.`

I'm not familiar with SevenShores. Should this notice remain as it is or should it be updated to something like the vendor/name of this repository?

**Reply by ryanwinchester** [reply]
Just change it to vendor or repo or whatever
